app/assets/stylesheets/utilities/layout.scss

```diff
@@ -42,3 +42,7 @@
 .health-check-unhealthy {
   background: red;
 }
+
+.unavailable-ip {
+  color: $govuk-error-colour;
+}
```

Comment:
The css is currently split between `/utilities` and `/components`. I would suggest creating css classes such as `error` and `red` in a file `utilities/colours.scss`

Reply:
Yeah, this was very much a chucked-in-not-sure-how-to-arrange-this-css job... might take the time to quickly clear this up, think this particular one should be in `/components` (since it's describing a particular part of the `ip/table` "component")

Reply:
Think 4be35c makes this a lot better.